frontend/src/pages/AdminPanel.js

```diff
@@ -0,0 +1,1253 @@
+    
+      optionsToAdd.forEach((new_option) => {
+        console.log(`POST Payload for Ordering Option:`, { new_option });
+        axiosClient
+          .post(`/kebabs/${kebabId}/ordering-options`, { new_option })
+          .then(() => console.log(`Opcja zamówienia "${new_option}" dodana pomyślnie.`))
+          .catch((error) =>
+            console.error(`Błąd przy dodawaniu opcji zamówienia "${new_option}":`, error)
+          );
+      });
+    
+      optionsToRemove.forEach((option_to_remove) => {
+        console.log(`DELETE Payload for Ordering Option:`, { option_to_remove });
+        axiosClient
+          .delete(`/kebabs/${kebabId}/ordering-options`, { data: { option_to_remove } })
+          .then(() => console.log(`Opcja zamówienia "${option_to_remove}" usunięta pomyślnie.`))
+          .catch((error) =>
+            console.error(`Błąd przy usuwaniu opcji zamówienia "${option_to_remove}":`, error)
+          );
+      });
+    }
+
+    if (JSON.stringify(localPages) !== JSON.stringify(initialKebab.pages)) {
+      const pagesToAdd = localPages.filter((page) => !initialKebab.pages?.includes(page));
+      const pagesToRemove = initialKebab.pages?.filter((page) => !localPages.includes(page));
+  
+      pagesToAdd.forEach((new_page) => {
+          axiosClient
+              .post(`/kebabs/${kebabId}/pages`, { new_page })
+              .then(() => console.log(`Dodano stronę: ${new_page}`))
+              .catch((error) => console.error(`Błąd przy dodawaniu strony:`, error));
+      });
+  
+      pagesToRemove.forEach((page_to_remove) => {
+          axiosClient
+              .delete(`/kebabs/${kebabId}/pages`, { data: { page_to_remove } })
+              .then(() => console.log(`Usunięto stronę: ${page_to_remove}`))
+              .catch((error) => console.error(`Błąd przy usuwaniu strony:`, error));
+      });
+    }
+
+    setIsKebabModalOpen(false);
+  };
+  
+  const openKebabDeleteConfirm = (kebab) => {
+    setSelectedKebab(kebab);
+    setIsKebabDeleteConfirmOpen(true);
+  };
+  
+  const closeKebabDeleteConfirm = () => {
+    setSelectedKebab(null);
+    setIsKebabDeleteConfirmOpen(false);
+  };
+  
+  const deleteKebab = () => {
+    axiosClient
+      .delete(`/kebabs/${selectedKebab.id}`)
+      .then(() => {
+        setKebabs((prevKebabs) =>
+          prevKebabs.filter((kebab) => kebab.id !== selectedKebab.id)
+        );
+        closeKebabDeleteConfirm();
+      })
+      .catch((error) => {
+        console.error('Error deleting kebab:', error);
+      });
+  };
+  
+  {/* Suggestions API */}
+  const [suggestions, setSuggestions] = useState([]);
+  useEffect(() => {
+    if (activeTab === 'suggestions') {
+      axiosClient
+        .get('/suggestions')
+        .then((response) => {
+          if (response.data && Array.isArray(response.data.suggestions)) {
+            setSuggestions(response.data.suggestions);
+          } else {
+            console.error('Unexpected response format:', response.data);
```

Comment:
Sometimes we got polish language in code and sometimes we got english. It's not a big issue, but it would be clearer to do it in one language

Reply:
Polish language is because our page is in Polish, but we're coding in English so..

Reply:
I mean, notifications are sometimes in different languages
![image](https://github.com/user-attachments/assets/9522f2f6-0d71-4beb-92b0-6b9296330944)

Reply:
it's just in console bruh after it's all will be done console logs can be removed